Detectors/FIT/FT0/workflow/src/ReconstructionSpec.cxx

```diff
@@ -35,10 +36,12 @@ void ReconstructionDPL::init(InitContext& ic)
 
 void ReconstructionDPL::run(ProcessingContext& pc)
 {
+  auto& mCCDBManager = o2::ccdb::BasicCCDBManager::instance();
+  mCCDBManager.setURL("http://ccdb-test.cern.ch:8080");
+  LOG(INFO) << " set-up CCDB";
```

Comment:
yes, this should be ok, though will change once the support of the CCDB by the DPL will be in place